trunk/SUAVE/Plots/Geometry_Plots.py

```diff
@@ -0,0 +1,152 @@
+## @ingroup Plots
+# Geometry_Plots.py
+# 
+# Created:  Mar 2020, M. Clarke
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+import SUAVE
+from SUAVE.Core import Units 
+import matplotlib.cm as cm
+import matplotlib.pyplot as plt  
+import numpy as np
+from mpl_toolkits.mplot3d import Axes3D
+from mpl_toolkits.mplot3d.art3d import Poly3DCollection
+from SUAVE.Methods.Geometry.Two_Dimensional.Cross_Section.Airfoil.import_airfoil_geometry \
+     import import_airfoil_geometry
+
+## @ingroup Plots
+# ------------------------------------------------------------------
+# Plot Airfol
+# ------------------------------------------------------------------
+def plot_airfoil(airfoil_names,  line_color = 'k-', save_figure = False, save_filename = "Airfoil_Geometry"):
+	"""This plots all airfoil defined in the list "airfoil_names" 
+
+	Assumptions:
+	None
+
+	Source:
+	None
+
+	Inputs:
+	airfoil_geometry_files   <list of strings>
+
+	Outputs: 
+	Plots
+
+	Properties Used:
+	N/A	
+	"""
+	# get airfoil coordinate geometry     
+	airfoil_data = import_airfoil_geometry(airfoil_names)       
+
+	for i in range(len(airfoil_names)):
+		# separate x and y coordinates 
+		airfoil_x  = airfoil_data.x_coordinates[i] 
+		airfoil_y  = airfoil_data.y_coordinates[i]    
+
+		name = save_filename + '_' + str(i)
+		fig  = plt.figure(name)
+		axes = fig.add_subplot(1,1,1)
+		axes.set_title(airfoil_names[i])
+		axes.plot(airfoil_x, airfoil_y , line_color )                  
+		#axes.set_aspect('equal')
+		axes.axis('equal')
+		if save_figure:
+			plt.savefig(name +".png")          
+
+	return
+
+# ------------------------------------------------------------------
+#   Propeller Geoemtry 
+# ------------------------------------------------------------------
+def plot_propeller_geometry(prop, line_color = 'bo-', save_figure = False, save_filename = "Propeller_Geometry"):
+	"""This plots the geoemtry of a propeller or rotor
+
+	Assumptions:
+	None
+
+	Source:
+	None
+
+	Inputs:
+	prop
```

Comment:
be more specific, a propeller: SUAVE.Components.Energy....

Reply:
Done